deployer/hub.py

```diff
@@ -24,9 +28,10 @@ class Cluster:
     A single k8s cluster we can deploy to
     """
 
-    def __init__(self, spec):
+    def __init__(self, spec, config_path):
         self.spec = spec
-        self.hubs = [Hub(self, hub_yaml) for hub_yaml in self.spec["hubs"]]
+        self.config_path = config_path
+        self.hubs = [Hub(self, hub_yaml, config_path) for hub_yaml in self.spec["hubs"]]
```

Comment:
Mentioning `hub_yaml` is a bit misleading here I think, since has already been loaded as a dictionary from a YAML string. The distinction I'm making is that YAML implies the content is a string, but it doesn't appear to be a valid YAML string but a Python dictionary.

```suggestion
        self.hubs = [Hub(self, hub_spec, config_path) for hub_spec in self.spec["hubs"]]
```